Multisig/UI/App/RemoteNotificationHandler.swift

```diff
@@ -13,8 +13,8 @@ import Firebase
 
 class RemoteNotificationHandler {
 
-    @UserDefault(key: "io.gnosis.multisig.deviceID")
-    private var storedDeviceID: String?
+    @UserDefaultWithDefault(key: "io.gnosis.multisig.deviceID", defaultValue: UUID().uuidString)
```

Comment:
I am concerned about this current code. This new default value must be saved on the app startup, otherwise, it will not be persisted between app starts, and we will have different UUID for the same app / token. I think there's a bug here.

Reply:
I am not sure what do you mean? The first time we will try to send a register request it will be created and stored between sessions. Default value is taken only when nothing is stored.
Or did you have smth else in mind?

Reply:
I think it is better to set UUID explicitly on the first app launch instead of lazy saving it

Reply:
ok